.github/workflows/ci.yml

```diff
@@ -30,6 +30,8 @@ jobs:
           - "--no-default-features"
           # Default features
           - ""
+          # All features
+          - "--all-features"
```

Comment:
`trace`, `debug` including others were actually not being tested by the CI

I think opting for an `--all-features` CI job makes sense here as a general solution, solving issues that may arise on a case-by-case basis